packages/api/test/e2e/api.create.e2e.ts

```diff
@@ -27,6 +28,7 @@ describe('e2e api create', () => {
 
   beforeAll(async () => {
     await cryptoWaitReady();
+    await waitReady();
```

Comment:
do we need both?
do users need to add this in their code?

Reply:
According to polkadot docs, it just needs one of them... however keyring was still not ready... I will play with this a bit and modify..

Reply:
what I observed is... 
for all our other test we create api instance first via 
`````
api = await initApiPromise();
`````
and then create the accounts with keyring
````
alice = keyring.addFromUri('//Alice');
````
Here api creation internally waits for waitReady from wasm-crypto and hence we don't need to add waitReady for others...

However in api creation test... we create api instance later and hence this was causing issue...

Have created the accounts after api instance is created in a test..